OpenRA.Mods.Common/Traits/SupportPowers/GrantUpgradePower.cs

```diff
@@ -26,7 +26,7 @@ class GrantUpgradePowerInfo : SupportPowerInfo
 		[Desc("Duration of the upgrade (in ticks). Set to 0 for a permanent upgrade.")]
 		public readonly int Duration = 0;
 
-		[Desc("Cells")]
+		[Desc("Cells - affects whole cells only")]
 		public readonly int Range = 1;
```

Comment:
It works on cells instead of exactly circular ranges. That is why I added to the description.
```
 X 
XXX
 X 
```
Changing this would be either misleading or affect its behaviour.